mobile/mobile WatchKit Extension/SensorView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  SensorView.swift
+//  mobile WatchKit Extension
+//
+//  Created by Collin Prather on 3/20/20.
+//  Copyright © 2020 Collin Prather. All rights reserved.
+//
+
+import SwiftUI
```

Comment:
lol guessing this is a WIP?

Reply:
lol 😂

It actually all was fully functional, but I decided (at least rn) there was not a need for multiple screens on the watch app. Even though I ported over all the functionality here into different parts of the app, I didn't want to lose it all in it's original state. Probably time to clean this up.